package.json

```diff
@@ -49,6 +49,7 @@
   },
   "homepage": "https://github.com/SU-SWS/decanter-react#readme",
   "dependencies": {
+    "@heroicons/react": "^1.0.4",
```

Comment:
Could we remove the old package react-hero-icon now that we have this new one? Or are there other icons that still use the old package? 😃